sql/testdata/timedate

```diff
@@ -0,0 +1,38 @@
+statement ok
```

Comment:
True, but so do the directories `cli` and `client`. Optimizing for file completion isn't terribly compelling to me.